chromadb/test/test_api.py

```diff
@@ -1290,6 +1319,8 @@ def test_get_nearest_neighbors_where_n_results_more_than_element(api):
     for key in results.keys():
         if key in includes or key == "ids":
             assert len(results[key][0]) == 2
+        elif key == "included":
```

Comment:
oh for future reference, this is really a grab bag of tests, we try to pull tests out on-demand, we can leave for now